app/script/localization/strings-cs.coffee

```diff
@@ -439,6 +439,7 @@ z.string.cs.preferences_options_data_detail = 'Přispějte k vylepšení aplikac
 z.string.cs.preferences_options_notifications = 'Upozornění'
 z.string.cs.preferences_options_notifications_none = 'Vypnout'
 z.string.cs.preferences_options_notifications_obfuscate = 'Skrýt podrobnosti zprávy'
+z.string.cs.preferences_options_notifications_obfuscate_message = 'TODO: Show only sender'
```

Comment:
No need to add the placeholders. Our translations are handled via Crowdin (https://crowdin.com/projects/wire). We mostly rely on the support of our users. A few of the languages are mostly translated by internal people. We always fall back to the English version if a translated version is not present, thus placeholders are contra productive. Please remove. I will ensure that the new string is present on Crowdin, when we are sure this is ready to be merged.